source/dotnet/Library/AdaptiveCards/AdaptiveAction.cs

```diff
@@ -49,5 +49,25 @@ public abstract class AdaptiveAction : AdaptiveTypedElement
 #endif
         [DefaultValue("default")]
         public string Style { get; set; } = "default";
+
+        /// <summary>
+        ///    When set false, action is disabled 
+        /// </summary>
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+#if !NETSTANDARD1_3
+        [XmlAttribute]
+#endif
+        [DefaultValue(true)]
+        public bool IsEnabled{ get; set; } = true;
```

Comment:
did you mean to include this in this PR?

Reply:
you might have seen it before my rebasing ...